### DISCUSSION TITLE
vCenter

### DISCUSSION THREAD

**Discussion by bouzeghoub** [Security-Onion-Solutions/securityonion#3381] [Unsupported Versions] [answered]
Hi,

Please, could you support me about vCenter port miror architect.

Best regards.

**Comment by launchdaemon** [comment, marked as answer]
Do you mean mirroring traffic to a port on a virtual switch in vCenter?  This might help:

https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.networking.doc/GUID-68B5DD45-DD3F-4E9B-A6CD-BE97026A846A.html

**Reply by bouzeghoub** [reply]
Thank you,
About a best architecte on vCenter.

**Reply by launchdaemon** [reply]
That's a very broad question.  Is there something specific you are trying to do?

**Reply by bouzeghoub** [reply]
I am sorry,
I think a solution is create mirror port on physical switch where is mapped on switch virtual after create mirror port on vCenter.
Thank you so much.
Best regards.
